elastic/integrations
# Removing detection rules from the `lmd` package

## What does this PR do?
This PR moves detection rules associated with Lateral Movement Detection (lmd) out of the integration package. 

## Checklist

## How to test this PR locally
- Spin up a local Elasticsearch cluster
- Install the Lateral Movement Detection assets
- Verify that no detection rules with the `Lateral Movement Detection` tag show up on the Rules Management page under Security

## Related Issues
Not linking since these are internal issues

## Screenshots
- Package updated to version `2.0.0`
<img width="898" alt="Screen Shot 2023-09-08 at 9 48 00 AM" src="https://github.com/elastic/integrations/assets/30438249/84853cdd-c797-47ea-9947-b5e2d7e4e92f">

- Job IDs have been updated to reflect the changes in this PR
<img width="1096" alt="Screen Shot 2023-09-08 at 9 56 55 AM" src="https://github.com/elastic/integrations/assets/30438249/d0d5d9f2-4d80-4efb-83e5-a60f8f140014">

- No rules have been installed upon installing the package assets
<img width="1640" alt="Screen Shot 2023-09-08 at 9 58 33 AM" src="https://github.com/elastic/integrations/assets/30438249/f0ec7c1b-67f7-4aea-9c50-56cd03553923">
